### ISSUE TITLE
Accessibility features in each Account screen

### ISSUE THREAD

**Issue by jco898** [AppDevOrganization/5A6-group-project#62]
**Task Description**
The different Account screens need accessibility features for the Cards, indicating the type of account and balance. The buttons need to indicate what clicking on them will do. Each transaction should be read out loud with all details included.

**Steps**
- Add custom click labels for the _Transfer_ and _Pay_ buttons.
- Ensure these buttons have a minimum touch target size of 48dp.
- Add custom merging to Cards to read out the account types and their totals.
- Ensure that clicking on any transaction will read out all details together, rather than individually.

**Technical effort (LOE)**
8
